### DISCUSSION TITLE
Welcome to wal-cli Discussions!

### DISCUSSION THREAD

**Discussion by Essbante** [roots-id/wal-cli#2] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by blaggacao** [comment]
Is it possible to think about federation with https://spiffe.io/?

IOG SRE would be interested in using DiDs to derive (human!) SVID-JWT & SVID-x509 for all sorts of identity & AuthC requirements.
